x-pack/solutions/security/plugins/security_solution/server/lib/siem_migrations/rules/task/agent/sub_graphs/translate_rule/nodes/translation_result/translation_result.ts

```diff
@@ -13,13 +13,16 @@ import { RuleTranslationResult } from '../../../../../../../../../../common/siem
 import type { GraphNode } from '../../types';
 
 export const getTranslationResultNode = (): GraphNode => {
-  return async (state) => {
+  return async (state, config) => {
     // Set defaults
     const elasticRule = {
       title: state.original_rule.title,
       description: state.original_rule.description || state.original_rule.title,
       severity: DEFAULT_TRANSLATION_SEVERITY,
       risk_score: DEFAULT_TRANSLATION_RISK_SCORE,
+      prebuilt_rule_id: config.configurable?.skipPrebuiltRulesMatching
+        ? null
+        : state.elastic_rule.prebuilt_rule_id,
```

Comment:
Doesn't it work if we just set `null` here, and we let the next line override it if defined?

Reply:
Your suggestion is working fine.. Fixed in https://github.com/elastic/kibana/pull/222542/commits/14528cdb1bc6f715673a0961f3d5801007be9cdd

Reply:
Yes i also just found that the setting just `prebuilt-rule-id` is not enough since other attributes remain unchanged as can be seen in below document and it is an invalid state

![image](https://github.com/user-attachments/assets/bd925898-943f-4fd9-9b5f-6215ee3f159a)


What do you think of setting `elastic_rule` as `null` as below  in defaults based on config: 

![image](https://github.com/user-attachments/assets/dcf7b0dc-8879-4677-a4e1-1628a5432a1d)

https://github.com/elastic/kibana/blob/71006bd7bda530fe4758759492386161d214ad21/x-pack/solutions/security/plugins/security_solution/server/lib/siem_migrations/rules/task/agent/sub_graphs/translate_rule/nodes/translation_result/translation_result.ts#L17-L20

Reply:
here are changes for nullifying complete object : https://github.com/elastic/kibana/pull/222542/commits/06c92d714f1545a057dabcd44db6e9acd10528a4